libraries/ms-helper/inc/thermistor.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+// Thermistor Reading interface for NXRT15XH103FA1B (10k Ohm NTC)
+// Requires ADC, GPIO and Interrupts to be initialized.
+#include "adc.h"
+
+typedef struct {
+  uint32_t sibling_resistance;  // milliohms
```

Comment:
Please choose a more obvious name than `sibling_resistance` and use ohms for user input. We'll never have a resistor divider that's in the mOhm range.

By this name it's unclear what resistor this should be in the resistor divider. It would be better to support both types. (i.e. thermistor is R1 vs. R2)

Reply:
Sure, I can change the code to support both types. And for the resistors, most of the are in the kOhm range right?

Reply:
Yes, but ohms is probably fine. Using kOhms wouldn't allow us to use things like 5.3kOhm.